### PR TITLE
For #258: Implemented Logs all(), stodout() and stderr()

### PR DESCRIPTION
**Pull request by bkuzmic**
For #258:

- Implemented Logs.all(), Logs.stdout() and Logs.stderr().

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#273](https://github.com/amihaiemil/docker-java-api/issues/273) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#273](https://github.com/amihaiemil/docker-java-api/issues/273) is assigned to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), here is [why](https://www.0crat.com/footprint/G6LPQQV2P/85b00598-731b-4f34-96af-b0cd88a261a2); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @amihaiemil[/z](https://www.0crat.com/u/amihaiemil) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

File: src/main/java/com/amihaiemil/docker/Logs.java [hunk 1]
```diff
@@ -53,6 +53,33 @@
...
     Reader follow() throws IOException, UnexpectedResponseException;
+
+    /**
+     * Show both stdout and stderr logs.
+     * @return New Logs instance.
+     * @throws IOException If something goes wrong.
+     * @throws UnexpectedResponseException If the status response is not
+     *  the expected one.
+     */
+    Logs all() throws IOException, UnexpectedResponseException;
```
Comment (amihaiemil) [inline src/main/java/com/amihaiemil/docker/Logs.java]: @bkuzmic Actually, I think the ``all()`` method is not needed. Let's read them all by default and if they want only ``stderr`` or ``stdout`` then they call the respective methods :D

File: src/main/java/com/amihaiemil/docker/RtLogs.java [hunk 2]
```diff
@@ -65,6 +60,16 @@
...
+
+    /**
+     * Flag for stdout option.
+     */
+    private final boolean stdout;
+
+    /**
+     * Flag for stderr option.
+     */
+    private final boolean stderr;
```
Comment (amihaiemil) [inline src/main/java/com/amihaiemil/docker/RtLogs.java]: @bkuzmic Can you leave a puzzle so we find a more elegant solution than having them as class attributes? There are more params for logs, so maybe it should be a ``Map`` or something.

**Comment by amihaiemil** [comment]
@rultor merge it

**Comment by 0crat** [comment]
The job [#273](https://github.com/amihaiemil/docker-java-api/issues/273) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil)

**Pull request merged**